routes/error-tracker.js

```diff
@@ -49,6 +49,51 @@ function ignoreMessageOrException(message, exception) {
 }
 
 /**
+ * @desc converts stack traces and standardizes them to chrome like.
+ * @param {string} exception
+ * @return {string} standardized exception
+ */
+function stackTraceConversion(exception) {
+  let chromeStackTraceRegex = /^\s*at (?:([^]*) )?([^]+):(\d+):(\d+)/m;
+  let match = chromeStackTraceRegex.test(exception);
+  if (match) {
+    exception = exception.substring(exception.indexOf('\n'));
+    let exceptions = exception.split('\n');
+    let validExceptions = exceptions.filter(function(value) {
+      return chromeStackTraceRegex.test(value);
+    });
+    exception = validExceptions.join('\n');
+    return exception;
+  } else if (!match) {
+    let mozillaSafariStackTraceRegex = /^([^@]*)@(.+):(\d+):(\d+)$/m;
+    let otherMatch = mozillaSafariStackTraceRegex.test(exception);
+    if (otherMatch) {
+      // convert to chromeLike
+      let exceptions = exception.split('\n');
+      let usableExceptions = exceptions.filter(function(value) {
+        return mozillaSafariStackTraceRegex.test(value);
+      });
+      let validExceptions = usableExceptions.map(safariOrMozillaToChrome);
```

Comment:
We already have all the information from the regex run above, we could just map there.